sunny-famnit:0.1.0

Tiggax opened the pull request:
<!--
Thanks for submitting a package! Please read and follow the submission guidelines detailed in the repository's README and check the boxes below. Please name your PR as `name:version` of the submitted package.

If you want to make a PR for something other than a package submission, just delete all this and make a plain PR.
-->

I am submitting
- [x] a new package
- [ ] an update for a package

This is a thesis template for University of Primorska, 
Faculty of Mathematics, Natural Sciences and Information Technologies


Description: Explain what the package does and why it's useful.

It is a thesis template for fast and easy final work project.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

Tiggax commented:
Follow-up question:
- Is `famnit` as a template package too ambiguous?

laurmaedje commented on packages/preview/famnit/0.1.0/template/main.typ:
```diff
@@ -0,0 +1,49 @@
+#import "@preview/famnit:0.1.0": project
+
+#show project.with(
```
There are a few syntax errors in this file.

Tiggax replied:
Forgot the commas :sweat_smile: . I fixed them

laurmaedje reviewed (commented):
The name `famnit` is indeed not permissible as it is exactly the name of the faculty. We want to have a level playing field where another person from FAMNIT could also submit a template. For templates, we recommend to combine a unique adjective with the descriptive part. See the [naming section of the submission guidelines](https://github.com/typst/packages?tab=readme-ov-file#submission-guidelines) for details.

Tiggax commented:
I've some problems I missed the first time. 
And I also updated the template to use the new context.

I renamed the package to be `sunny-famnit`. I hope this is a good enough name. :sweat_smile: 

If there aren't any more problems i think it should be good to go :smile:
